servor.js

```diff
@@ -7,6 +7,13 @@ const os = require('os');
 const net = require('net');
 const cwd = process.cwd();
 
+const watch = (path, cb) => {
+  if (fs.statSync(path).isDirectory()) {
+    fs.watch(path, cb);
+    fs.readdirSync(path).forEach(entry => watch(`${path}/${entry}`, cb));
```

Comment:
I feel like this might become too expensive. I use `servor` at the root of repository, which has lot of directories in `.git`, `node_modules`, etc.

maybe we should consider being able to configure this, something like:
```bash
servor --watch dist --watch src <...other options>
```

Reply:
I have considered this too! I'm not sure exactly how expensive this is.. but apparently the native `{ recursive: true }` is much more efficient (to the point where you can do:

```
// watch the whole disk
fs.watch('/', { recursive: true })
```

So I am going to put a ternary in that will use this for linux and native recursive fs.watch on windows and mac (then we shouldn't see any regression on existing perf).

Regards you wanting to watch only certain things I'm playing with the idea of a config file to put all these harder to define inline kind of options in. As I have mentioned before I want to keep the CLI and node API as simple as possible.

Reply:
One more alternative you might want to consider:

<img width="1111" alt="Screen Shot 2020-01-30 at 2 58 51 PM" src="https://user-images.githubusercontent.com/802242/73498105-26d3c680-4371-11ea-9d8a-c3367663c17a.png">

Although this involves much more complicated options forwarding through CLI.

Reply:
I'd rather keep options to to a minimum as branching like this creates more and more cases that require testing etc. I have been on vacation the past week and only just getting getting a chance to go through GitHub notifications. Will push the native/custom watch switch change asap. Thanks for your patience @thisguychris and @osdevisnot 🙇

Reply:
I have pushed the change now so @osdevisnot you should not notice any difference in the way servor works and @thisguychris it should still work for you on linux! 🤞

Reply:
@lukejacksonn confirming that it works, more details on #36